run_dir/static/js/ont_trend_plot.js

```diff
@@ -133,6 +106,38 @@ function make_plot(key, name, filter_inst_type, color_type, plot_type){
         }
     };
 
+    if (key == "basecalled_pass_bases") {
+        toplot.tooltip.formatter = function () {
+                const name = this.point.name;
+                const color = this.point.color;
+                const series_name = this.series.name || '';
+                const y_value = typeof this.y !== 'undefined' ? this.y : '';
+                const passed_reads = typeof this.point.passed_reads !== 'undefined' ? format_value(this.point.passed_reads) : '';
+                const sample_name = this.point.sample_name || '';
+    
+                return `<span style="color: ${color}">\u25CF</span><small>${name}</small><br />
+                    <b>${series_name}</b>: ${y_value}<br>
+                    <b>Basecalled Pass Read Count:</b> ${passed_reads}<br>
+                    <b>Sample Name:</b> ${sample_name}`;
+        }
+    }
+
+    if (key == "basecalled_pass_read_count") {
+        toplot.tooltip.formatter = function () {
+            const name = this.point.name;
+            const color = this.point.color;
+            const series_name = this.series.name || '';
+            const y_value = typeof this.y !== 'undefined' ? this.y : '';
+            const passed_bases = typeof this.point.passed_bases !== 'undefined' ? format_value(this.point.passed_bases) : '';
```

Comment:
Could we have passed_reads here as well, and passed bases correspondingly within the basecalled_pass_bases above? I think it would be good.